matching/score.go

```diff
@@ -43,7 +42,7 @@ func Score(places []Place, distNorm int) float64 {
 func singlePlaceScore(place Place) float64 {
 	var ratingPricingRatio float64
 	if place.PlacePrice() == 0 {
-		ratingPricingRatio = AvgRating / AvgPricing // set to average single Place rating-price ratio
+		ratingPricingRatio = float64(place.Rating()) / MaxPlaceRating
```

Comment:
nit: we can rename `ratingPricingRatio` to `boostFactor`, since when the place price is zero, it is more like a normalized rating.

Reply:
Done.